main/src/main/java/cgeo/geocaching/unifiedmap/geoitemlayer/MapsforgeVtmGeoItemLayer.java

```diff
@@ -123,14 +123,14 @@ private int getZLevel(final GeoPrimitive item) {
     public Pair<Drawable, MarkerInterface> add(final GeoPrimitive item) {
 
         final int fillColor = GeoStyle.getFillColor(item.getStyle());
-        final float rawStrokeWidth = GeoStyle.getStrokeWidth(item.getStyle());
+        final float rawStrokeWidth = GeoStyle.getStrokeWidth(item.getStyle()) / (item.getType() == GeoItem.GeoType.POLYGON ? 1 : 2);
```

Comment:
This one I really don't like. I don't see this fix so urgently needed that we should apply a hack. We should take the time to investigate what is happening further up in the code and apply a real fix.

Reply:
Our fist goal should be that geoitems are rendered consistent for all map types. If we need a factor to get consistent rendering it's totally fine. But 1 distance unit should be the same width for all map implementations and geoitem types.

Reply:
>  If we need a factor to get consistent rendering it's totally fine. But 1 distance unit should be rendered as the same width for all map implementations and geoitem types.

Consistency is exactly what this PR implements with the conditional factor.

Screenshots show a styled geojson (Duisburg), a unstyled geojson (Cologne) and a route on unified/old OSM/Google Maps.
There's still a slight different between Unified Google/OSM, but that's due to dropDistance() only being applied to OSM (should it be applied to Google as well?)

![Screenshot_20240215_201502](https://github.com/cgeo/cgeo/assets/1258173/2a28616e-c81c-46e6-9151-c5a497681280)
![Screenshot_20240215_202355](https://github.com/cgeo/cgeo/assets/1258173/fc67b79e-c864-44b1-9d75-9246f7337486)
![Screenshot_20240215_201644](https://github.com/cgeo/cgeo/assets/1258173/672f151b-ae8b-4139-a64d-67da996a1649)
![Screenshot_20240215_201510](https://github.com/cgeo/cgeo/assets/1258173/a4967e2e-f59f-4cc3-b889-e32fb2c2b88b)

Reply:
Yes but it is a hack to implement consistency exactly in the reported bug case - and will most likely cause inconsistentcy in other cases (needs to be throughly tested).

The Geo layer is designed such that any width in a GeoStyle should be strictly interpreted as dp. That means:
* any code using the Geo layer needs to make sure that given widths are in do
* An implementation of a Geo layer for a specific map provider (VTM, Google, mapsforge) needs to make sure to convert given values such that it is drawn in the wanted dp width

An implementation using different factors for different shapes (poly line vs polygon) can't possibly achieve this consistency.

I know that VTM is more tricky than plain Google API, but let's please find consistent solutions.

Reply:
> and will most likely cause inconsistentcy in other cases (needs to be throughly tested).

Do you have anything in mind not yet covered by the screenshots above? I can check it then.
Before this PR Geojson rendering is inconsistent VTM Google vs VTM OSM vs old maps. I'm trying to make them look the same everywhere and this is the best fit for all the cases i could think off.

Applying a random factor is certainly not nice (why i called it "ugly" in my original comment), but i couldn't find any better place to fix it to get the same result across all maps. Seems the pipeline to get things into Google vs OSM is still quite different, even in VTM.

Reply:
I would suggest to do the following:

Draw a polyline next to a polygon using the same line width. You might use the `GeoItemTestLayer `class to do so (I created it for the purpose to try out things like that). Then view the result across Google and VTM and check:
* is the line width rendered the same for both objects?
* is the line width rendered the same for both map types?
* does line width behave as expected when zooming in or out?

If you find that different factors for different types (polygon, polyline) need to be used in VTM then your solution is the correct fix. Because the different geoitemlayer-implementations are then consistent to each other.

If not then the fault lies on the input side, although I could not explain why line width would differ across map types in this case.

Reply:
> Draw a polyline next to a polygon using the same line width. You might use the `GeoItemTestLayer `class to do so (I created it for the purpose to try out things like that). Then view the result across Google and VTM and check:

Did that - and you're right, polygon and polyline width are different now (with the different factors). However I realized that all elements but the polygon are to thin on OSM vs Google:
![Screenshot_20240216_091312](https://github.com/cgeo/cgeo/assets/1258173/e358dd7d-54d9-4cc4-94bb-82f7c6227e15)
![Screenshot_20240216_091344](https://github.com/cgeo/cgeo/assets/1258173/22696d77-a6d3-4b19-b41d-33815df7ee10)

So I changed the factor from 2f to 1.5f globally and am now getting a consistent rendering OSM vs Google vs Old:
![Screenshot_20240216_091944](https://github.com/cgeo/cgeo/assets/1258173/b28a5805-7b34-49b6-9ada-34deaff55462)
![Screenshot_20240216_091912](https://github.com/cgeo/cgeo/assets/1258173/2c29043c-2085-4c19-876f-c8cb0ea9a0b4)

BUT .. the navigation/history lines are now wrong: 75 (OSM) vs 63 (Google). And this is what I compared against initially (why I introduced the second factor). So I guess that's the real issue that needs to be fixed in regards to the factor.

I'll update this PR with the change of the factor from 2f to 1.5f, leaving the wide OSM navigation line as the todo-item (because I don't have the time to look into that today).